tests/test_api.py

```diff
@@ -294,3 +297,61 @@ def test_url_protocol_association(delete_files):
         url_to_open=url,
         expected_output=url,
     )
+
+
+@pytest.mark.skipif(PLATFORM != "win", reason="Windows only")
+def test_windows_terminal_profiles():
+    def _parse_terminal_profiles(settings_file: Path) -> dict:
+        settings = json.loads(settings_file.read_text())
+        return {
+            profile.get("name", ""): profile.get("commandline", "")
+            for profile in settings["profiles"]["list"]
+        }
+
+    settings_files = [
+        # Stable
+        Path(
+            folder_path("user", False, "localappdata"),
+            "Packages",
+            "Microsoft.WindowsTerminal_8wekyb3d8bbwe",
+            "LocalState",
+            "settings.json",
+        ),
+        # Preview
+        Path(
+            folder_path("user", False, "localappdata"),
+            "Packages",
+            "Microsoft.WindowsTerminalPreview_8wekyb3d8bbwe",
+            "LocalState",
+            "settings.json",
+        ),
+        # Unpackaged (Scoop, Chocolatey, etc.)
+        Path(
+            folder_path("user", False, "localappdata"),
+            "Microsoft",
+            "Windows Terminal",
+            "settings.json",
+        ),
+    ]
+    settings_files = [file for file in settings_files if file.exists()]
+    if not settings_files:
+        pytest.skip("No terminal profile settings file found.")
+    tmpdir = mkdtemp()
+    (Path(tmpdir) / ".nonadmin").touch()
```

Comment:
```suggestion
    (tmp_path / ".nonadmin").touch()
```

Let's use the `tmp_path` fixture instead.

Reply:
Update instances below too 🙏